### DISCUSSION TITLE
Kindle for Mac, Calibre & noDRM/DeDRM version recommendations

### DISCUSSION THREAD

**Discussion by romad275** [noDRM/DeDRM_tools#360] [Q&A]
OK, I've been using Kindle for Mac 1.17, Calibre 3.48, & DeDRM 6.6.2 on my Mid-2011 iMac which is maxed out at MacOS 10.13.6 High Sierra. I have a 15" MacBook Pro that is maxed out at MacOS 12.6.x Monterey and I'm considering moving my ebook processing to it. 

So I'd like to know which are the best versions of Kindle for Mac, Calibre, and DRM plug-in to use with MacOS 12.6.x on the MacBook Pro.

Dennis B. Swaney

**Comment by ElleKayEm** [comment]
You obviously can run the latest calibre which is 6.22 right now.  For that, you'll need DeDRM 10.0.3.  Kindle for Mac is more complicated.  If you're decrypting books published before 2023, you can use Kindle for Mac 1.25 through 1.31 by disabling KFX downloads and preventing updates to the app.  See method 2B at this post:  https://www.mobileread.com/forums/showthread.php?t=283371
And this Mac info:  https://www.mobileread.com/forums/showpost.php?p=3819708&postcount=508

For books published this year, method 3 (E-Ink Kindle) and method 6 (Kindle for Android) allow you to avoid KFX and get books which can be decrypted with the release version of DeDRM.

Recently someone has figured out how to decrypt the newer KFX DRM (in most cases...at least for the moment.)  Their version is here:  https://github.com/Satsuoni/DeDRM_tools/releases/tag/v10.0.5s

noDRM has added that code to the master code here, but hasn't put out a release with that update yet.  You can get it from this page though:  https://github.com/noDRM/DeDRM_tools/actions/runs/5370838206  (Click "plugin" at the bottom there.  You'll need to unzip it twice.)

So with either of those, you should be able to decrypt books from the current Kindle for Mac 1.40.  You'll also need the KFX plugin (https://www.mobileread.com/forums/showthread.php?t=291290).
